plugins/janus_videoroom.c

```diff
@@ -7505,6 +7546,14 @@ static void janus_videoroom_recorder_close(janus_videoroom_publisher *participan
 		janus_recorder *rc = participant->vrc;
 		participant->vrc = NULL;
 		janus_recorder_close(rc);
+		// upload file
+		// upload_recording(rc->filename,rc->codec);
+		// try watch
+		printf("[file debug]------calling create_recording_json\n");
+
+		int res = create_recording_json(rc->filename, rc->codec);
+		printf("[file debug]------res = %d\n", res);
```

Comment:
Does printf go to wgw logs? cloud watch?

Reply:
yes